src/content/docs/full-stack-observability/observe-everything/get-started/new-relic-guided-install-overview.mdx

```diff
@@ -0,0 +1,150 @@
+---
+title: New Relic guided install overview
+tags:
+  - Full-Stack Observability
+  - Observe everything
+  - Get started
+contentType: page
+template: basicDoc
+metaDescription: Our guided installation is a single command that you can use to discover and start monitoring integrations on your system. 
+---
+
+Instrument your systems and send telemetry data to New Relic with guided install.
+
+<ButtonLink
+  role="button"
+  to="https://one.newrelic.com/launcher/nr1-core.home?pane=eyJuZXJkbGV0SWQiOiJucjEtY29yZS5ob21lLXNjcmVlbiJ9&cards[0]=eyJuZXJkbGV0SWQiOiJucjEtaW5zdGFsbC1uZXdyZWxpYy5ucjEtaW5zdGFsbC1uZXdyZWxpYyIsImFjdGlvbkluZGV4IjoxfQ=="
+  variant="primary"
+>
+  Guided install
+</ButtonLink>
+
+If you're hosted in the EU:
+
```

Comment:
I'm not a fan of how these render on the site. If you're going to have the "If you're hosted in the eu" line, I'd do a similar line for the other button.